packages/core/src/browser/context-menu-renderer.ts

```diff
@@ -29,6 +29,14 @@ export function toAnchor(anchor: HTMLElement | Coordinate): Anchor {
     return anchor instanceof HTMLElement ? { x: anchor.offsetLeft, y: anchor.offsetTop } : anchor;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isAnchor(arg: any): boolean {
+    if (arg && 'x' in arg && 'y' in arg && Object.keys(arg).length === 2) {
```

Comment:
Suggestion: We try to limit our `any` usage in the framework.
```suggestion
export function isAnchor(arg: unknown): boolean {
    if (typeof arg === 'object' && arg && 'x' in arg && 'y' in arg && Object.keys(arg).length === 2) {
```